src/main/java/seedu/address/ui/AppointmentCard.java

```diff
@@ -33,50 +34,40 @@ public class AppointmentCard extends UiPart<Region> {
     @FXML
     private Label appointmentDateTime;
     @FXML
-    private Label hasAttended;
-    @FXML
     private Label appointmentDescription;
     @FXML
-    private Label feedbackScore;
+    private FlowPane details;
 
     /**
      * Creates a {@code AppointmentCode} with the given {@code Appointment} to display.
      */
-    public AppointmentCard(Appointment appointment) {
+    public AppointmentCard(Appointment appointment, String patientName) {
         super(FXML);
         this.appointment = appointment;
 
         appointmentId.setText(appointment.getAppointmentId() + ". ");
-        //TODO: replace student id with student name
-        name.setText("StudentId: " + Integer.toString(appointment.getPatientId().patientId));
+        name.setText(patientName);
 
         String formattedStartDateTime = DateUtil.formatDateTime(appointment.getStartDateTime().getDateTimeValue());
         String formattedEndDateTime = DateUtil.formatDateTime(appointment.getEndDateTime().getDateTimeValue());
-        appointmentDateTime.setText("Appointment Time: from " + formattedStartDateTime + " \nto "
+        appointmentDateTime.setText(formattedStartDateTime + " to "
```

Comment:
can change to `-` for cleaner UI

Reply:
Noted. It is now 
![image](https://github.com/AY2324S2-CS2103-F08-2/tp/assets/86278971/19c92466-226d-4a5a-8a89-e8c09d035b7d)